test/e2e/actors.http.test.ts

```diff
@@ -48,10 +49,26 @@ describe('http/actors', () => {
   }, 30 * 1000);
 
   afterAll(async () => {
-    // await server.stop();
+    await server.stop(); // if we hang here, it means connections are open that were not closed. Debug why
     // await client.stop();
   });
 
+  describe('general', () => {
+    it('should allow us to use promises as some testing libraries have issues within the Node Ecosystem', async () => {
```

Comment:
1. Should we retain this test?
2. If yes, should this be a part of actor test suite?

Reply:
Will remove